recipes-core/cvm-reverse-proxy-client/files/cvm-reverse-proxy-client-init

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          cvm-reverse-proxy-client
+# Required-Start:    $network $remote_fs $syslog
+# Required-Stop:     $network $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Start CVM Reverse Proxy Client at boot time
+# Description:       Enable CVM Reverse Proxy Client service.
+### END INIT INFO
+
+DAEMON=/usr/bin/proxy-client
+NAME=cvm-reverse-proxy-client
+DESC="CVM Reverse Proxy Client"
+DAEMON_ARGS="--listen-addr=localhost:7947 --target-addr=http://localhost:7937 --client-attestation-type azure-tdx --server-attestation-type none"
```

Comment:
that one does the client-side aTLS, right?

in this case, per the port list, it should
- listen on port 7937
- forward to https://hub-atls.builder.flashbots.net